src/mtxLora/commands/uplink/MtxCommand.ts

```diff
@@ -0,0 +1,36 @@
+import MtxCommandBase from '../../MtxCommandBase.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../../constants/directions.js';
+
+
+/**
+ * Uplink command.
+ *
+ * @example
+ * ```js
+ * import MtxCommand from 'jooby-codec/mtxLora/commands/uplink/MtxCommand.js';
+ *
+ * const command = new MtxCommand({
```

Comment:
uplink example should be reversed

hex -> params

Reply:
fixed